CHANGELOG.md

```diff
@@ -2,13 +2,38 @@
 
 ## Table of Contents
 
+- [v0.4.0-rc.2](#v040-rc2)
 - [v0.4.0-rc.1](#v040-rc1)
 - [v0.4.0-alpha.1](#v040-alpha1)
 - [v0.3.0](#v030)
 - [v0.2.0](#v020)
 - [v0.1.0 - Initial version](#v010---initial-version)
 
 
+# v0.4.0-rc.2
+
+**This is the second Release Candidate version for Commonalities release 0.4.0.**
+
+It contains the following corrections compared to [v0.4.0-rc.1](#v040-rc1).
```

Comment:
Would be needed to add https://github.com/camaraproject/Commonalities/pull/267 once merged.

LGTM in advance